CHANGELOG.md

```diff
@@ -1,7 +1,30 @@
-## Next release
-
-* [ENHANCEMENT] Allow overridability of the web page title (#3525)
-* [BUGFIX] set pkg/apis/monitoring tag to v0.42.1 (#3531)
+## 0.43.0 / 2020-10-26
+
+This release introduces a new `AlertmanagerConfig` CRD that allows to split the
+Alertmanager configuration in different objects. For now the CRD only supports
+the PagerDuty, OpsGenie and webhook receivers, [other
+integrations](https://github.com/prometheus-operator/prometheus-operator/issues?q=is%3Aissue+is%3Aopen+%22receiver+type%22)
+will follow in future releases of the operator. The current version of the CRD
+is `v1alpha1` meaning that the feature is not yet considered stable and the API
+is subject to change in the future.
+
+* [CHANGE] Use a single reloader sidecar (instead of 2) for Prometheus. The `--config-reloader-image` flag is deprecated and will be removed in a future release (not before v0.45.0). Make sure to start the operator with a version of `--prometheus-config-reloader` that is at least `v0.43.0` otherwise the Prometheus pods will fail to start. #3457
```

Comment:
Should we make a **warning** here in bold that its going to be deprecated, I always notice a lot of people just glance at this?

Reply:
done